Add log drains addons (#1007)
Scalingo/documentation: _posts/platform/app/2000-01-01-log-drain.md

* `host`: host of the syslog server
* `port`: port number of the syslog server

### Add a Drain to an Addon

> To add a log drain to an addon you have to pass its uuid. First you have to list
> them with the command to find addon's uuid:

Reviewer: ```suggestion
In order to add a log drain to an addon, you need to get its UUID. List your application addons to get its UUID:

```

```bash
scalingo --app my-app addons
```

Then, use the option `--addon <uuid>` with the [add command](#add-a-log-drain):
```bash
scalingo --app my-app log-drains-add --addon ad-a86f546c-6b08-439b-a5fc-6f16ddc9083e \
  --type syslog --host custom.logstash.com --port 12345
```

> Currently, only database addons are able to forward logs to a drain.

Reviewer: This is not completely true. All addons are able to forward logs to a drain. It's just that non-database addons do not generate logs, at all.

> Currently, only databases addons are able to forward logs to a drain.

Reviewer: ```suggestion
Currently, only database addons are able to forward logs to a drain.
```

+--------------------------------------------------------------------+
```

> You can also list drains from both application and connected addons with the

Reviewer: ```suggestion
You can also list drains from both the application and the associated addons with the
```

```bash
scalingo --app my-app log-drains-remove tcp+tls://logs.papertrailapp.com:10333
```

{% note %}
By default the log drain will be removed from both the application and the

> connected addons. To remove a drain from only the application you can use

Reviewer: ```suggestion
related addons. To remove a drain only from the application you can use
```
